coffee_core/src/coffee.rs

```diff
@@ -201,8 +201,17 @@ impl CoffeeManager {
         let path = self.config.cln_config_path.clone().unwrap();
         let mut file = CLNConf::new(path.clone(), true);
         log::info!("looking for the cln config: {path}");
-        file.parse()
-            .map_err(|err| CoffeeError::new(err.core, &err.cause))?;
+        file.parse().map_err(|err| {
+            CoffeeError::new(
+                err.core,
+                format!(
+                    "coffee was unable to locate Core Lightning configuration file at the specified path: {}.
+                     To resolve this, please run the following command: coffee setup <cln_path>.",
+                    path
+                )
+                .as_str(),
+            )
```

Comment:
Mh I do think this is always right, if the file is malformed an error is returned, and in this way you are hiding it

Reply:
Do you prefer to keep the previous error and instead check if the path exists separately and return this coffee error if it's not found?

Reply:
What is the solution that we should recommend to the user?

Reply:
What I am saying that the parse function will not fails only if the file is missing, but will return an error also when the configuration file is not good

the parse function will fails if you put the following content

```
[bitcoin]
plugin=/path/bin
```

You return `coffee was unable to locate Core Lightning configuration file` but this is not true

Reply:
> but will return an error also when the configuration file is not good

Ah yes I totally missed this case 
Thanks!